index.js

```diff
@@ -0,0 +1,66 @@
+document.addEventListener('DOMContentLoaded', () => {
+    getFilm()
+
+    let review = document.getElementById("review");
+    let list = document.getElementById("list");
+    let form = document.querySelector("form");
+
+    form.addEventListener("submit", (event) => {
+        event.preventDefault();
```

Comment:
This is not a bad of adding Event listener for "submit" action but another way of improving this function would be to put li creation and append to another callback function. It is recommended to make more organized and readable. Good work with this though.